hack/docs/metrics_gen/main.go

```diff
@@ -39,6 +40,16 @@ type metricInfo struct {
 	help      string
 }
 
+var (
+	stableMetrics = []string{"controller_runtime", "aws_sdk_go", "client_go", "leader_election", "interruption", "cluster_state", "workqueue", "karpenter_build_info", "karpenter_nodepool_usage", "karpenter_nodepool_limit",
```

Comment:
Can we think about doing this through markers above the metric declarations themselves? Something similar to how kubebuilder does marker parsing for CRD generation